client/ClientNetworking.cpp

```diff
@@ -487,8 +505,14 @@ void ClientNetworking::Impl::SendSelfMessage(Message&& message) {
 
 boost::optional<Message> ClientNetworking::Impl::GetMessage() {
     auto message = m_incoming_messages.PopFront();
-    if (message)
-        TraceLogger(network) << "ClientNetworking::GetMessage() : received message " << *message;
+    if (message) {
+        if ((*message).Compressed()) {
```

Comment:
can you do `message->Compressed()` ? (I'm not sure, but would be nicer...)

Reply:
Change made in https://github.com/freeorion/freeorion/pull/3854/commits/4ead6795ce2b51bce839955a679750840ada4831.